lightning/src/ln/channelmanager.rs

```diff
@@ -2873,9 +2872,11 @@ const MAX_NO_CHANNEL_PEERS: usize = 250;
 /// short-lived, while anything with a greater expiration is considered long-lived.
 ///
 /// Using [`ChannelManager::create_offer_builder`] or [`ChannelManager::create_refund_builder`],
-/// will included a [`BlindedMessagePath`] created using:
-/// - [`MessageRouter::create_compact_blinded_paths`] when short-lived, and
-/// - [`MessageRouter::create_blinded_paths`] when long-lived.
+/// will included a [`BlindedMessagePath`] created using [`MessageRouter::create_blinded_paths`].
+///
+/// To use compact [`BlindedMessagePath`] one can use [`ChannelManager::create_offer_builder_using_router`]
+/// and [`ChannelManager::create_refund_builder_using_router`] to pass a custom router for compact
+/// blinded path creation.
```

Comment:
Technically, a user could parameterize `ChannelManager` with `CompactMessageRouter` to achieve the same thing. Though that would mean reply paths (when created from `ChannelManager`, not `OnionMessenger`) would also use the compact format, which while is not wrong is probably not necessary.

Reply:
Makes sense!
Updated the documentation to reflect both the methods in [pr3246.18](https://github.com/shaavan/rust-lightning/commits/pr3246.18)
Thanks!